### PR TITLE
Added method which converts signatureDate to UTC timezone

### LINKED ISSUE #1329: set custom timezone for simulator in ui tests
Of course it is a timezone issue. Can you set in appium runner to run the iOS simulator on UTC timezone regardless of the timezone of the host? There must be settings for it.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/pull/1321#discussion_r789875899_

### PR DESCRIPTION
**Pull request by fcvakintos**
This PR contains changes for checkSignatureDateValue method, convert date to UTC timezone
close #1329 

----------------------------------

**Tests** 
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: appium/tests/specs/mock/setup/UpdateOlderPubKeyToNewer.spec.ts [hunk 1]
```diff
@@ -40,6 +40,9 @@ describe('SETUP: ', () => {
     const newFingerprintsValue = CommonData.updateRecipientPublicKey.newFingerprints;
 
     await mockApi.withMockedApis(async () => {
+      const times = await driver.getDeviceTime();
+      console.log('times', times);
```
Comment (sosnovsky) [inline appium/tests/specs/mock/setup/UpdateOlderPubKeyToNewer.spec.ts:44]: Is it needed or was just for debug purpose?

Comment (fcvakintos) [reply appium/tests/specs/mock/setup/UpdateOlderPubKeyToNewer.spec.ts:44]: yes, i will remove it, sorry

Comment (fcvakintos) [reply appium/tests/specs/mock/setup/UpdateOlderPubKeyToNewer.spec.ts]: @sosnovsky can we skip  test run after this update?

Comment (sosnovsky) [reply appium/tests/specs/mock/setup/UpdateOlderPubKeyToNewer.spec.ts]: Why skip it? I think it should pass without any issues

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**